src/sagemaker/local/entities.py

```diff
@@ -38,6 +38,135 @@
 HEALTH_CHECK_TIMEOUT_LIMIT = 120
 
 
+class _LocalProcessingJob(object):
+    """Placeholder docstring"""
```

Comment:
Could you fill in all the docstrings in this class?

Reply:
Added docstrings and integration tests.